setup.sh

```diff
@@ -78,6 +164,23 @@ for i in ${!other_challenges[@]}; do
     read -p "What is the GitHub username of your teammate doing challenge #${other_challenges[i]}? " team[i]
 done
 
+
+if [ "${team[0]}" == "${team[1]}" ]; then
+    printf "\e[31mTeam members can not have the same GitHub username\e[0m\n"
```

Comment:
Nice!

Reply:
Can combine this with the other conditions below by `if [ ... ] || [ ... ] || [ ... ]`